src/bionics.cpp

```diff
@@ -1278,6 +1278,21 @@ bool player::uninstall_bionic(std::string const &b_id)
         return false;
     }
 
+    // surgery is imminent, retract claws or blade if active
+    if (has_bionic("bio_claws")) {
+        if (weapon.type->id == "bio_claws_weapon") {
+            add_msg(m_neutral, ("You withdraw your claws."));
```

Comment:
Ahhh, so close. The `(` and `)` are there, but the `_` is missing. That means: No translation for you!